lib/plugins/inspec-parallel/lib/inspec-parallel/runner.rb

```diff
@@ -72,24 +83,40 @@ def spawn_another_process
 
         child_reader, parent_writer = IO.pipe
 
-        # Construct command-line invocation
+        error_log_file = nil
         child_pid = nil
-        error_log_file_name = "#{Time.now.nsec}.err"
-
         begin
+          logs_dir_path = log_path || Dir.pwd
+          log_dir = File.join(logs_dir_path, "logs")
+          FileUtils.mkdir_p(log_dir) unless File.directory?(log_dir)
+          error_log_file = File.open("#{log_dir}/#{Time.now.nsec}.err", "a+")
           cmd = "#{$0} #{sub_cmd} #{invocation}"
           log_msg = "#{Time.now.iso8601} Start Time: #{Time.now}\n#{Time.now.iso8601} Arguments: #{invocation}\n"
-          child_pid = Process.spawn(cmd, out: parent_writer, err: error_log_file_name)
-          # Rename error log file if exist
-          rename_error_log(error_log_file_name, child_pid) if File.exist?(error_log_file_name)
+          child_pid = Process.spawn(cmd, out: parent_writer, err: error_log_file.path)
+
+          # Trap Control-c Interrupts
+          trap("SIGINT") do
+            # Kill child process instantly
+            Process.kill("SIGINT", child_pid)
```

Comment:
This fix does not seem to be working for me. When I interrupt the process using ctrl+c, it hangs on there.